### ISSUE TITLE
Cameras buffering heavily

### ISSUE THREAD

**Issue by paulharken** [dwainscheeren/dwains-lovelace-dashboard#379]
Hello Dwain

Love your dashboard, great piece of work. I have a brief question and don't know if anyone else has experienced it. On a Lenovo 7" tablet, when you click on a camera feed to open it up, the RTSP streams do not render without buffering constantly. However, the exact entity on the normal HA LoveLace dashboard I have created renders perfectly, extremely fast with no buffering.

This issue is solely isolated, from what I can find, to your dashboard. Do you parse cameras feeds/streams differently to compared to the stock HA lovelace dashboards? Is there anything I can do to try diagnose the issue?

I'm using the [Unifi Protect](https://github.com/briis/unifiprotect) plugin. The streams I have tried are both 1280x720 and 640x360. Buffering still occurs (although slightly less) on the 640 image.

**Comment by dwainscheeren** [comment]
How many cameras do you got added in dd? I also got the unifi protect integration myself with 4 cameras and it works without problems.

**Comment by paulharken** [comment]
Thank you for the very prompt reply!

Interesting. I'm glad we have the same platform. 4 cameras as well. One of the streams worked this afternoon, albeit briefly. When I loaded up another one, it went back to buffering. 

The only reason I've queried if it's your dashboard is because it only happens in there on the tablet. The tablet itself is a Lenovo M7 third generation, so it was made last year. Pretty decent specs but I'm wondering if there's something to do with the RTSP stream from Ubiquiti.
